src/kaleidoscope/driver/led/WS2812.h

```diff
@@ -33,8 +33,10 @@
 
 #pragma once
 
-#include "kaleidoscope/hardware/avr/pins_and_ports.h"
+#ifndef KALEIDOSCOPE_VIRTUAL_BUILD
```

Comment:
virtual build guards should get pulled to their own commits.

Reply:
I don't recall why I added this guard here.  I'll try simply removing it and see what happens.  It might be necessary to make the new header include changes work, though, and if that's true, I think it belongs in the same commit as those header changes.

Reply:
The simulator tests run just fine without this, so it might be that it's helpful for IWYU, but I'd rather not go searching for why I had it in there right now, so I'm planning to leave it out.